<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>120</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>120</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>120</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BMS TOOL V3.35 KEYGEN</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/BMS.ico</normaloff>:/BMS.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="lblCode">
      <property name="text">
       <string>LOGIN CODE</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>349</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnCalculate">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Calculate</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="leCode">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="leToken">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>12</number>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lblToken">
      <property name="text">
       <string>MAC</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>COMPANY</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="cbCompany">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <item>
       <property name="text">
        <string>拓邦 (TAKUHO, ADMINISTRATOR)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>海雷 (SEA RAY, CLIENT)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>聚亚科技 (JUYA TECHNOLOGY,  SUPPLIER)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>奥闻科技 (AOWEN TECHNOLOGY, SUPPLIER)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>移远 (MOVE AWAY, SUPPLIER)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>固勤科技 (GUQIN TECHNOLOGY, SUPPLIER)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>恩步通信 (ONSTEP COMMUNICATION, SUPPLIER)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>博实结科技 (BOSHIJIE TECHNOLOGY, SUPPLIER)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>拓邦锂电 (TOPBAND LITHIUM BATTERY, CLIENT)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>康普盾 (COMPTON, CLIENT)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>菏泽天盈 (HEZE TIANYING, CLIENT)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>联动天翼 (LINKAGE TIANYI, CLIENT)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>江苏智泰 (JIANGSU ZHITAI, CLIENT)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>博磊达 (BRIGHTA, CLIENT)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>芯讯通 (SIMCOM, SUPPLIER)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>公共默认公司 (PUBLIC DEFAULT COMPANY, SUPPLIER)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>公共测试公司 (PUBLIC TESTING COMPANY, SUPPLIER)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>南都 (NANDU, CLIENT)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>蜂巢能源 (HIVE ENERGY, CLIENT)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>新能安 (XINNENGAN, CLIENT)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>天能 (TIANNENG, CLIENT)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>亿伟 (YIWEI, CLIENT)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>孚能 (FUNENG, CLIENT)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>沃泰通 (WOTAITONG, CLIENT)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>宁德时代 (CATL, CLIENT)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>上海果下科技 (SHANGHAI GUOXIA TECHNOLOGY, CLIENT)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>智租 (SMART RENT, CLIENT)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>星恒 (STELLAR, CLIENT)</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
